test/gurka/test_phonemes.cc

```diff
@@ -1252,23 +1230,20 @@ TEST(Standalone, PhonemesWithAltandDirection) {
       } else
         FAIL() << std::get<kLinguisticMapTuplePronunciationIndex>(iter->second)
                << " Extra key. This should not happen.";
-
-      ++sign_index;
     }
 
     // more signs.  jeita wins.
     node_id = BC_edge->endnode();
     tile = graph_reader.GetGraphTile(node_id);
-    node_info = tile->node(node_id);
+    tile->node(node_id);
```

Comment:
same here

Reply:
and all other places below

Reply:
If I remove the lines, everything fails, that is why initially I kept the [[Maybe_unused]] on the node_info